clusterDeployer.py

```diff
@@ -303,6 +303,7 @@ def need_external_network(self) -> bool:
                (len(self._cc["workers"]) > len(self._cc.worker_vms()))
 
     def deploy(self) -> None:
+        nfs.export(self._iso_path)
```

Comment:
Needs more work. This is an incomplete fix. We need to have a way to ask files to be hosted in an NFS directory, and, if the server has yet to be started, then start it.

Reply:
I will work on the proper fix. But for now, new deployments need this fix to proceed.